python/voxel_mapper.py

```diff
@@ -5,18 +5,43 @@
 import  numpy as np
 import numbers
 import argparse
-
+from io import BytesIO
+from nibabel import FileHolder, Nifti1Image
+from gzip import GzipFile
+from hdfs import Config
 
         
 def mapper(bin_size, data_max, data_min):
     for line in sys.stdin:
             line = line.strip()
             line = line.split()
             slice_file = os.path.join(line[0])
-            
 
-            #open nifti image
-            slice = nib.load(slice_file)
+            #if image is located in hdfs of not (ie stored locally)
+            hdfs_file = True if line[1] == "True" else False
```

Comment:
And i suppose this wont be a problem at all once i figure out how to pass hdfs URLs